### PR TITLE
Update concepts-architecture-overview.md

### PR DESCRIPTION
**Pull request by PiotrJander**
Unfinished sentence - how should it be completed?

*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: site2/docs/concepts-architecture-overview.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ In a Pulsar cluster:
 
 * One or more brokers handles and load balances incoming messages from producers, dispatches messages to consumers, communicates with the Pulsar configuration store to handle various coordination tasks, stores messages in BookKeeper instances (aka bookies), relies on a cluster-specific ZooKeeper cluster for certain tasks, and more.
 * A BookKeeper cluster consisting of one or more bookies handles [persistent storage](#persistent-storage) of messages.
-* A ZooKeeper cluster specific to that cluster handles
+* A ZooKeeper cluster specific to that cluster handles ???
```
Comment (sijie) [inline site2/docs/concepts-architecture-overview.md]: service discovery and metadata operations.

**Comment by codelipenghui** [comment]
@PiotrJander Any update on this PR? I move this PR to 2.7.0 first. If there are any updates on this PR, I will move it back.

**Comment by Jennifer88huang** [comment]
@PiotrJander Thank you very much for your contribution. I check and find that the information is updated since v2.6.0 release. Before v2.6.0, the content is not complete. Could you update it as "A ZooKeeper cluster specific to that cluster handles coordination tasks between Pulsar clusters."? 

@sijie If we adopt the [doc version control solution](https://github.com/apache/pulsar/issues/7413) on this, it might be better. When the contributor pull request, we can add "release/x.x.x" to adopt the changes into related versions.

**Comment by sijia-w** [comment]
The pr to fix the conflicts is https://github.com/apache/pulsar/pull/7597.

**Comment by Jennifer88huang** [comment]
@PiotrJander thanks very much for your contribution.
Since it's fixed in #7597, we can close this PR.

**Pull request closed without merging**